src/config/theme.ts

```diff
@@ -511,6 +512,17 @@ const createThemeOptions = (theme: Theme) => ({
             },
           }),
         },
+        {
+          // styles for gray alert
+          props: { color: 'gray' },
+          style: theme.unstable_sx({
+            borderColor: theme.palette.grayscale[300],
+            backgroundColor: theme.palette.grayscale.surface,
+            '& .MuiAlert-icon': {
+              color: theme.palette.grayscale.main,
```

Comment:
I'd like to move towards the more succinct language for specifying colors, `'grayscale.300'`, `'grayscale.surface'` etc